eng/pipelines/runtime.yml

```diff
@@ -415,630 +453,630 @@ jobs:
           eq(dependencies.checkout.outputs['SetPathVars_mono.containsChange'], true),
           eq(variables['isFullMatrix'], true))
       # extra steps, run tests
-      extraStepsTemplate: /eng/pipelines/common/templates/runtimes/wasm-runtime-and-send-to-helix.yml
+      extraStepsTemplate: /eng/pipelines/common/templates/runtimes/android-runtime-and-send-to-helix.yml
```

Comment:
instead of introducing an extra step template, use a shared template for both wasm and android, maybe name it `mobile-*` and then based on the `osGroup` parameter, condition the steps and parameters accordingly.

Reply:
Will clean it up in a follow-up PR.